data/announcements/scheduled.yaml

```diff
@@ -130,3 +130,25 @@ announcements:
   message: |
     5 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     October 24 - 28, 2022.
+
+- name: Freezing k8s.gcr.io - Before
+  startTime: 2023-02-20T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
+                                 # This should run before and after Kubecon EU 2023
+  endTime: 2023-04-02T00:00:00
```

Comment:
there's another PR open to update this with another announcement from 3/27 to 4/4. I'd change it to end on 3/26 and we can put it back on 4/5 to run through kubecon eu

Reply:
Fixed

Reply:
thanks!